### ISSUE TITLE
Add check for images not linking images

### ISSUE THREAD

**Issue by ivanbranco** [osm-fr/osmose-backend#2173]
Hi, do you think it would be possible to add a check for images not linking images? The wiki for image=* says there are three different styles:

- Wikimedia Commons filename (formatted as File:image.jpg) - although this is commonly tagged as [wikimedia_commons](https://wiki.openstreetmap.org/wiki/Key:wikimedia_commons)=*
- [ URL](https://en.wikipedia.org/wiki/en:URL) or [ URI](https://en.wikipedia.org/wiki/en:Uniform_Resource_Identifier) of the image itself
- URL or URI of a page containing the image alongside copyright information and other details

So if an image value is neither a URL or starting with File: it could be flagged to be checked and fixed. Examples of elements I would expect to see flagged:

- https://www.openstreetmap.org/node/9952657599 - in this case a user put the subject=* value as image=* value, no image is linked.
- https://www.openstreetmap.org/node/11270510002 - not a link, not a commons file
- https://www.openstreetmap.org/way/815564696 - coordinates
- ecc.ecc.

**Comment by frodrigo** [comment]
Yes, interesting. If you think it may done with MapCSS, it is probably better to do it in JOSM MapCSS files.
